commcare_connect/audit/chc_indicators.py

```
@@ -0,0 +1,518 @@
+        return coverage_ratio / visit_ratio, total_eligible
+
+
+@register_calculation
+class InaccessibleWARateEarlyWarning(AuditCalculation):
+    """Detect emerging overuse of Inaccessible state mid-WAG.
+    Evaluated against the FLW's current active WAG.
+    Requires ≥5 WAs with terminal status before evaluating.
+    """
+
+    name = "inaccessible_wa_rate_early_warning"
+    label = "Inaccessible WA Rate – Early Warning"
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)